README.md

```diff
@@ -22,90 +22,57 @@
 
 ## About
 
-### `payjoin`
+`payjoin/rust-payjoin` contains multiple crates intended to assist the implementation of Payjoin functionality as defined in [BIP 78: Payjoin V1 (Synchronous)](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) and [BIP 77: Payjoin V2 (Asynchronous)](https://github.com/bitcoin/bips/blob/master/bip-0077.md), and the deployment of OHTTP relays and Payjoin directories which facilitate Async Payjoin transactions.
 
-The Payjoin Dev Kit `payjoin` library implements both [BIP 78 Payjoin V1](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) and [BIP 77 Payjoin V2](https://github.com/bitcoin/bips/blob/master/bip-0077.md).
+Find the description of each crate below.
 
-### `payjoin-cli`
+### [`payjoin`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin)
 
-The [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) crate performs no-frills Payjoin as a reference implementation using Bitcoin Core wallet.
+The main Payjoin Dev Kit library which provides tools for implementing both Payjoin V2 (Asynchronous) and V1 (Synchronous). For Payjoin V2, the library also contains Payjoin session persistence support and IO utilities for interacting with OHTTP relays.
```

Comment:
I've been doing everything I can to have people call them "Sync Payjoin" and "Async Payjoin" rather than v1/v2 which has proven confusing. Is it possible to change the language so that we just ignore v1/v2 and say e.g. "The main payjoin Dev Kit library which provides tools for implementing Async Payjoin and earlier protocols."